native/service/governance/neo3_state_manager/neo3_state_manager.go

```diff
@@ -0,0 +1,158 @@
+package neo3_state_manager
+
+import (
+	"fmt"
+	"github.com/polynetwork/poly/common"
+	"github.com/polynetwork/poly/native"
+	"github.com/polynetwork/poly/native/event"
+	"github.com/polynetwork/poly/native/service/governance/node_manager"
+	"github.com/polynetwork/poly/native/service/utils"
+)
+
+const (
+	//function name
+	GET_CURRENT_STATE_VALIDATOR      = "getCurrentStateValidator"
+	REGISTER_STATE_VALIDATOR         = "registerStateValidator"
+	APPROVE_REGISTER_STATE_VALIDATOR = "approveRegisterStateValidator"
+	REMOVE_STATE_VALIDATOR           = "removeStateValidator"
+	APPROVE_REMOVE_STATE_VALIDATOR   = "approveRemoveStateValidator"
+
+	//key prefix
+	STATE_VALIDATOR           = "stateValidator"
+	STATE_VALIDATOR_APPLY     = "stateValidatorApply"
+	STATE_VALIDATOR_REMOVE    = "stateValidatorRemove"
+	STATE_VALIDATOR_APPLY_ID  = "stateValidatorApplyID"
+	STATE_VALIDATOR_REMOVE_ID = "stateValidatorRemoveID"
+)
+
+// todo, review code
+
+func GetCurrentStateValidator(native *native.NativeService) ([]byte, error) {
+	svs, err := getStateValidators(native)
+	if err != nil {
+		return utils.BYTE_FALSE, fmt.Errorf("GetCurrentStateValidator, getStateValidators error: %v", err)
+	}
+	data := SerializeStringArray(svs)
```

Comment:
why this place need to serialize it？

Reply:
Since a `Handler` can only return []byte.

Reply:
```go
Handler func(native *NativeService) ([]byte, error)
```

Reply:
udpated